docs/how_it_works/ranking.rst

```diff
@@ -60,9 +60,9 @@ as shown in this plot below.
 
 .. nbimport::
```

Comment:
This code snippet contains an error (noticed it on main and fixed there, but we might as well apply it here:


```
# ranker fits on one metric and reference period data only

ranker2.fit(

    realized_perf_results.filter(period='reference', metrics=['recall']))

# ranker ranks on one drift method and one performance metric

correlation_ranked_features2 = ranker2.rank(

    univariate_results.filter(period='analysis', methods=['jensen_shannon']),

    realized_perf_results.filter(period='analysis', metrics=['recall']),

    only_drifting = False)
```